packages/core/src/modules/dcql/DcqlService.ts

```diff
@@ -714,28 +714,18 @@ export class DcqlService {
             throw new DcqlError('Cannot create presentation for credentials without subject id')
           }
 
-          // Determine a suitable verification method for the presentation
-          const verificationMethod = await this.getVerificationMethodForSubjectId(
-            agentContext,
-            presentationToCreate.subjectIds[0]
-          )
-
           const w3cV2CredentialService = agentContext.resolve(W3cV2CredentialService)
           const w3cV2Presentation = new W3cV2Presentation({
+            holder: presentationToCreate.credentialRecord.credential.resolvedCredential.credentialSchemaIds[0],
```

Comment:
This seems incorrect. It should not be `credentialSchemaIds` right? Because that's the id from `credentialSchema.id` (not from `credentialSubject.id`)